docs/languages/powershell.md

```diff
@@ -0,0 +1,128 @@
+
+You can also Open them with ctl/cmd+shift+p and typing __PowerShell: Open Examples Folder__.
+
+![Open PowerShell Examples](./images/powershell/pwshExamples.png)
+
+## Major Features
+
+* Syntax Highlighting
+* Code Snippets
+* Intellisense for cmdlets and more
+* Rule-Based analysis provided by PowerShell Script Analyzer
+* Go to definition of cmdlets and variables
+* Find references of cmdlets and variables
+* Document and Workspace symbol discovery
+* Run slected selection of PowerShell code using F8
+* Launch online help for the symbol under the cursor using ctl+F1
+* Local script debugging and basic interactive console support
+
+## Plaster
+
+ Plaster is a template-based file and project generator written in PowerShell. It's Purpose is to streamline the creation of PowerShell module projects, Pester tests, DSC configurations and more.
+ See the [Github Plaster repo](https://github.com/PowerShell/Plaster) for more information.
+
+ [Cmdlet Documentation](https://github.com/PowerShell/Plaster/blob/master/docs/en-US/Plaster.md)
+
+## Debugging
+
+Check out the ![HSG](./images/powershell/HSGavatar.png) Hey, Scripting Guy blog Posts two part series on debugging
+
+Written by [@keithHill](https://twitter.com/r_keith_hill)
+
+[Debugging PowerShell script in Visual Studio Code – Part 1](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/06/debugging-powershell-script-in-visual-studio-code-part-1/)
+
+[Debugging PowerShell script in Visual Studio Code – Part 2
+](https://blogs.technet.microsoft.com/heyscriptingguy/2017/02/13/debugging-powershell-script-in-visual-studio-code-part-2/)
+
+## Settings Reference
+
+You can customize VSCode settings from the file menu by selecting preferences then clicking settings. You can also click the gear Icon
+![codeGear](./images/powershell/codeGear.png) located in the lower left corner of the activity bar.
+
+This will launch the settings.json file, in VSCode versions prior to 1.27.1. , where the VSCode team has introduced a settings GUI, which will be the new default interface for customizing settings.
+
+You can still launch the JSON settings file by using the command __Open Settings (JSON)__ or by changing the default settings editor with the ```"workbench.settings.editor"```
+
+### VSCode Settings GUI
+
+![VSCode Setting GUI](./images/powershell/vscodeSettingsGui.png)
+
+### PowerShell Extension Settings
+
+![PowerShell Extension VSCode Settings](./images/powershell/pwshCodeSettings.png)
+
+[Using a specific version of powershell with VSCode](https://docs.microsoft.com/en-us/powershell/scripting/core-powershell/vscode/using-vscode?view=powershell-6#using-a-specific-installed-version-of-powershell)
+
+### PowerShell Extension Settings Categories
+
+* Project bug reporting settings
+* Code Folding
+* Code Formatting
+* Code Debugging
+* Developer
+* Integrated Console
+* Enable Profile Loading
+* Help Completion
+* PowerShell .exe specific settings
```

Comment:
Does this mean settings about what PowerShell executable to use or what arguments to pass to the executable? In either case, we should probably say "PowerShell executable settings", especially since `.exe` is a Windows-specific concept